examples/anthos-service-mesh-multicluster/vars.sh

```diff
@@ -0,0 +1,41 @@
+# Customize variables as needed
+
+# Terraform debug logging. 
+# Comment these two variables out if not running on local
+export TF_LOG="TRACE"
+export TF_LOG_PATH=~/terraform.log
```

Comment:
why write terraform log to home directory?

Reply:
Test lines, removing...